OAuth2: basic authorization when using refresh token (#7044)
hammerandchisel/discord-api-docs: docs/topics/OAuth2.md

}
```

Having the user's access token allows your application to make certain requests to the API on their behalf, restricted to whatever scopes were requested. `expires_in` is how long, in seconds, until the returned access token expires, allowing you to anticipate the expiration and refresh the token. 

> To refresh, make another `POST` request to the [token URL](#DOCS_TOPICS_OAUTH2/shared-resources-oauth2-urls) with basic authorization where `CLIENT_ID` is username, `CLIENT_SECRET` is password, and the following parameters:

Reviewer: ```suggestion
To refresh it, make another `POST` request to the [token URL](#DOCS_TOPICS_OAUTH2/shared-resources-oauth2-urls) with Basic authorization (`CLIENT_ID` as the username and `CLIENT_SECRET` as the password) plus the following parameters:
```

Reviewer 2: shouldn't `CLIENT_ID` and `CLIENT_SECRET` be lowercase too to match the rest of the page
